### PR TITLE
[UI] Replace header icons from FontAwesome to Remix

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
Replaced to the new icon pack.
Added the user's position above their name.

---
Reordered the notification icon to improve UX.

In the matter of information distribution, we easily realize that items on the right side of the screen are more related to technical management or administration. Either resources such as categories as well as the settings tab, having them all in one side tells users we have a consistency. 

So from this, we can think it would also make sense to keep the notification panel icon on the right side of the screen, since often it is responsible for showing information as a new update available. 

The left side, on the other hand, commonly presents items that marketers use in everyday work, like contacts, campaigns as well as other necessary resources for the success of a strategy. I believe that keeping only the search icon alone in this side improves consistency, since they will use global search to find these items easily.

## Before:
image

image

image

## After:

https://github.com/mautic/mautic/assets/116097999/238ff04d-63ee-4397-a818-c7a93c0332fb

image

image

image

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Look at the navbar

### CODE REVIEW

**Review by LordRembo — CHANGES_REQUESTED** [review]
I understand the purpose of these changes but my 1 suggestion would be to split this PR into 3 separate PR's. You are combining 3 changes that are not interdependent and some are a bigger change than others.

- moving the notification icon will be a point of contention and should be discussed before making a (separate) PR for it.
- showing the user position/function with the name is a nice improvement but has nothing to do with icons

**Comment by code5rick** [comment]
Love the animation that you made @andersonjeccel to show the user position!

File: app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig [hunk 1]
```diff
@@ -10,14 +10,15 @@
...
             </a>
         </li>
-        {{ render(controller('Mautic\\CoreBundle\\Controller\\DefaultController::notificationsAction')) }}
         {{ render(controller('Mautic\\CoreBundle\\Controller\\DefaultController::globalSearchAction')) }}
     </ul>
     <!--/ end: left nav -->
 
     <!-- start: right nav -->
     <ul class="nav navbar-nav navbar-right">
+
```
Comment (shinde-rahul) [inline app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig:19]:
```suggestion
        
```

**Review by shinde-rahul — CHANGES_REQUESTED** [review]
Looks good.

**Comment by andersonjeccel** [comment]
@LordRembo I'm thinking of the previous conversation about finding a balance between the difference/change size and testing efforts…

What if try this approach this time to see if people feel happy? If it goes wrong, I cherry pick these commits to another PR and separate them

File: app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig:19]: Hm, well noted! Thanks

**Comment by escopecz** [comment]
Some conflicts here too

**Comment by andersonjeccel** [comment]
@escopecz Solved!

**Comment by andersonjeccel** [comment]
@RCheesley Mapped these images on user docs:

translations-select-language
translations-select-user-language
mautic-report
need-to-enable-composer
marketplace-detail
dashboard-date-filters
gui-update-deprecated
notifications
asset_create
asset_settings
dwc_create
Mautic-31-company-view
social_monitoring_hashtags
sms_send_marketing_message
create_sms
social_monitoring_mentions
focus_item_create
new_segment_sms
new_template_sms

Notifications is the only/most relevant since it's about its location, is feasible to recapture others again now?

File: app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig:19]: _This commit didn't work bc GitHub says no changes were made to the code_

**Review by RCheesley — APPROVED** [review]
I *love* these changes. I think it makes total sense to move the notifications over to the right (I never knew why they were on the left) and the animation on the account is a nice addition.

One thing though, and I know I am being totally pedantic here, but I think it should maybe have a max width for the description after which maybe it should wrap - otherwise this happens: https://app.screencastify.com/v3/watch/CwBm7ATnYWvCc4jZypdl

That can be addressed in another PR though if folks think it needs fixing ;)

**Comment by RCheesley** [comment]
> Notifications is the only/most relevant since it's about its location, is feasible to recapture others again now?

@andersonjeccel I am not sure I fully understand the question here?

**Review by escopecz — APPROVED** [review]
I see no problem with the code changes 👍

**Comment by andersonjeccel** [comment]
@RCheesley I meant, all these listed images show header with the old location for the notifications icon (even when they’re not the focus)

but one image, “notifications” needs to be changed since it’s about where the user can click to find the panel

I'd like to know if we could leave the others (unrelated to notifications) as they are since capturing dozens of new images would take a long time to replace

**Comment by RCheesley** [comment]
Only need to update the images which are impacted by the change.

**Comment by RCheesley** [comment]
@andersonjeccel just a reminder that the docs are needed for this PR.

**Comment by andersonjeccel** [comment]
Had on my to-do list
Done!

**Pull request merged**